### PR TITLE
Add V

### PR DESCRIPTION
**Pull request by Delta456**
[V](https://vlang.io/) is a new programming language inspired from C, C++, Go and Python.

This list is very useful for those who want to know more about the language.

By submitting this pull request I confirm I've read and complied with the awesome list requirements 🖖

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by aliesbelik** [comment]
You've created a substantial collection of resources, but you appear to be missing a bunch of items from the requirements list.

Let's complete the quest:
- add 2 [PRs](https://github.com/sindresorhus/awesome/pulls) you reviewed;
- add `awesome-list` and `awesome` as GitHub topics (as well as any other relevant, if any);
- Awesome badge should be placed on the right side of the readme heading;
- ToC section should be named `Contents`, not `Table of Contents`;
- add license;
- add [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines);
- add project logo;
- finally check [guideline](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) guideline if something still missing.

**Comment by aliesbelik** [comment]
Please check list items formatting:
 - use dash `-` instead of asterisk `*` for the entries;
 - ensure that list item description start with valid casing;
 - use dash as separator between link and description.

Using [awesome-lint](https://github.com/sindresorhus/awesome-lint) may help you a lot.

**Comment by alexesprit** [comment]
@aliesbelik Hi! Thanks for reviewing `awesome-v`!

> use dash - instead of asterisk * for the entries;

Interesting that running `awesome-lint README.md` does not show errors if we use **asterisks only**.

File: readme.md [hunk 1]
```diff
@@ -217,6 +217,7 @@
 - [Q#](https://github.com/ebraminio/awesome-qsharp#readme) - Domain-specific programming language used for expressing quantum algorithms.
 - [Imba](https://github.com/koolamusic/awesome-imba#readme) - Programming language inspired by Ruby and Python and compiles to performant JavaScript.
 - [Vala](https://github.com/desiderantes/awesome-vala#readme) - Programming language designed to take full advantage of the GLib and GNOME ecosystems, while preserving the speed of C code.
+- [V](https://github.com/vlang/awesome-v) - Programming language inspired from C, C++, Go and Python.
```
Comment (alexesprit) [inline readme.md]: Should be at least "inspired by C, C++, Go and Python" or replaced with "official" desciption: "Simple, fast, safe, compiled language for developing maintainable software."

Comment (Delta456) [reply readme.md]: Ok, I will go with the official one.

File: readme.md [hunk 2]
```diff
@@ -217,7 +217,7 @@
 - [Q#](https://github.com/ebraminio/awesome-qsharp#readme) - Domain-specific programming language used for expressing quantum algorithms.
 - [Imba](https://github.com/koolamusic/awesome-imba#readme) - Programming language inspired by Ruby and Python and compiles to performant JavaScript.
 - [Vala](https://github.com/desiderantes/awesome-vala#readme) - Programming language designed to take full advantage of the GLib and GNOME ecosystems, while preserving the speed of C code.
-- [V](https://github.com/vlang/awesome-v) - Programming language inspired from C, C++, Go and Python.
+- [V](https://github.com/vlang/awesome-v) - Simple, fast, safe, compiled language for developing maintainable software.
```
Comment (alexesprit) [inline readme.md]: Also, you should add "#readme" anchor.

**Comment by aliesbelik** [comment]
> Interesting that running `awesome-lint README.md` does not show errors if we use **asterisks only**.

@alexesprit sorry, that's my fault: because of incorrect ToC header anchors in list entries were treated as incorrect URLs (`Invalid list item link URL`). Will be validated with using of `Contents` header instead.

**Comment by alexesprit** [comment]
Now `vlang/awesome-v` passes `awesome-lint`.

@aliesbelik What do you think? Have we some missing things should be fixed?

**Comment by aliesbelik** [comment]
Looks fine to me.

But to complete the quest please add at least 2 [PRs](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen) you reviewed for this project.
Dura lex, sed lex.

**Comment by alexesprit** [comment]
I'm not an author of this PR, but I want to get this one merged. Also, I contributed to `awesome-v` to make it possible to pass `awesome-lint`.

PRs I reviewed: #1675, #1679.

**Comment by alexesprit** [comment]
unicorn

**Comment by Delta456** [comment]
Reviewed https://github.com/sindresorhus/awesome/pull/1683 https://github.com/sindresorhus/awesome/pull/1682

**Comment by aliesbelik** [comment]
@alexesprit, I appreciate your efforts.

Looking into the other PRs I can note that adding link to V's website for `V` in `V is a simple, fast, safe, compiled language for developing maintainable software.` is nice to have.

All the other points look good to me.

**Comment by alexesprit** [comment]
@aliesbelik Good catch! I added the link to the V website.

**Comment by jetli** [comment]
Please use the default PR template of awesome project for your PR description, you can refer to https://github.com/sindresorhus/awesome/pull/1688, see the description of this PR.

**Comment by Delta456** [comment]
@jetli done

**Comment by FatemehTarashi** [comment]
Looks good. :+1:

**Comment by aliesbelik** [comment]
Seems like this PR can be finally merged.

**Comment by bernardoduarte** [comment]
Couldn't find anything wrong. Great awesome list!

**Comment by medvednikov** [comment]
Can you merge it please?

**Comment by alexesprit** [comment]
This PR should be rebased on to latest master to satisfy the guidelines:
> Your entry should be added at the bottom of the appropriate category.

Also, it breaks the Coq description - removes dot at the end of the description.
> The first character should be uppercase and the description should end in a dot.

**Comment by Delta456** [comment]
@medvednikov and @alexsprit done

**Comment by alexesprit** [comment]
It's still not at the bottom of the list.

**Comment by Delta456** [comment]
@alexesprit now done?

**Comment by medvednikov** [comment]
Can you merge please?

**Comment by Delta456** [comment]
Can anyone please merge this?

**Comment by medvednikov** [comment]
Any plans to merge?

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1254711189424099329

**Pull request merged**